### PR TITLE
Gkabbz mobile activation

### PR DESCRIPTION
**Pull request by gkabbz**
@magorlick Given the timeline to get some reporting to the SC next week, I'm thinking we stick with this query as it represents how the metric was written. It's not the most efficient of queries and we end up querying way less data if we used WAU instead to find the number of clients activated. But that would mean the metric is changing from day 2-7 to day 2-8.

### CODE REVIEW

File: src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+cohort as (
+SELECT
+  *
+FROM
+  last_seen_data
+WHERE
+-- filtering specifically for where the two dates match as the profile creation date sees more profiles created in days after the submission date i.e. Mar 6 has more client_ids with Mar 1 profile creation date than Mar 1
+-- Shouldn't be a material difference in the activation but should revalidate when productionizing
+  profile_date = cohort_date
+  AND submission_date = cohort_date
```
Comment (gkabbz) [inline src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql:46]: @magorlick Need to know what you think about this. I'm specifically filtering the cohort to users who submission_date = profile_date. This is to remove some of the profile_date noise and keep the cohorts steady. I was seeing users with a Mar 1 Profile date as early as Feb 15th. I realize it's not ideal but I don't think it changes the number in a material way.

File: src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+WITH last_seen_data as (
+SELECT
+  submission_date,
+  client_id,
+  days_since_seen,
+  days_since_created_profile,
+  profile_date,
+  app_name,
+  SPLIT(metadata_app_version,'.')[offset(0)] as app_version,
+  os,
```
Comment (gkabbz) [inline src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql:16]: @magorlick Not for this week but before we productionize next week. Let me know if there are any additional dimensions we'd like to add here.

File: src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+SELECT
+  *
+FROM
+  last_seen_data
+WHERE
+-- filtering specifically for where the two dates match as the profile creation date sees more profiles created in days after the submission date i.e. Mar 6 has more client_ids with Mar 1 profile creation date than Mar 1
+-- Shouldn't be a material difference in the activation but should revalidate when productionizing
+  profile_date = cohort_date
+  AND submission_date = cohort_date
+  AND product_name IN ('Fennec Android', 'Fenix', 'Focus Android', 'Fennec iOS', 'Focus iOS', 'Firefox Lite', 'FirefoxConnect', 'Lockwise Android')
```
Comment (magorlick) [inline src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql:47]: doesn't include Fenix yet.

File: src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  AND submission_date <= activation_period_end
+GROUP BY 1),
+
+activation as (
+SELECT
+  cohort.*,
+  usage.days_active
+FROM
+  cohort
+LEFT JOIN
```
Comment (magorlick) [inline src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql:67]: This method excludes those that we see a core ping in days 2-7 and not day 1.  Another method counts all unique clients days 1-7 (denominator) and unique clients days 2-7 (numerator).  Hard to know if we are biased in the other direction.  We should just pick a method and stick with it.

File: src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql (hunk 1 above)
Comment (magorlick) [reply src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql:46]: I think this sounds good.

**Pull request merged**